### PR TITLE
WEBOPS-912: ssh-key to hooks helper

### PR DESCRIPTION
**Pull request by DeviaVir**

### CODE REVIEW

**Comment by DeviaVir** [comment]
ping @lrvick @thepwagner

File: bin/ssh-key [hunk 1]
```diff
@@ -58,4 +58,7 @@ if [ $SKIP_KEY -eq 0 ]; then
 	git-shell-commands/backup
 fi
 
-rm -f $KEYFILE
\ No newline at end of file
+rm -f $KEYFILE
+
+cd /git
+[ ! -f "$hook_dir/post-ssh-key" ] || bash $hook_dir/post-ssh-key $@ | tee /var/log/git-deploy/hooks.log
```
Comment (thepwagner) [inline bin/ssh-key]: Confirming: this hook should fire even after a NOOP execution? (i.e. https://github.com/pebble/git-deploy/blob/e20346bad034c93e9a32da398d5929294e66714a/bin/ssh-key#L42 )

**Comment by thepwagner** [comment]
+1

File: bin/ssh-key (hunk 1 above)
Comment (DeviaVir) [inline bin/ssh-key]: thinking here was if we wanted to add a key to dynamodb that is already in `authorized_keys`; let our hooks run their logic regardless of what `git-deploy` thinks

**Pull request merged**